RELEASE.md

```diff
@@ -0,0 +1,88 @@
+Releases
+========
+
+This page describes the release processes. It's inspired from
+the [Prometheus release process](https://github.com/prometheus/prometheus/blob/main/RELEASE.md)
+
+## How to cut an individual release
+
+### Branch management and versioning strategy
+
+We use [Semantic Versioning](https://semver.org/).
+
+The current flow is to merge new features and bugfixes into the main branch.
+
+As we are in early stage of the project, we don't intend to maintain previous version. Only the last version will be
+patched if we estimate it requires an urgent patch that cannot wait for the next minor release.
+
+### 0. Updating dependencies
+
+We are using dependabot to keep up to date our dependencies (of Go or npm).
+
+In case you would like to update manually the dependencies :
+
+#### Updating Go dependencies
+
+```bash
+make update-go-deps
+git add go.mod go.sum
+git commit -m "Update Go dependencies"
+```
+
+### Updating npm dependencies
+
+```bash
+make update-npm-deps
+git add ./ui/package-lock.json ./**/package.json
+git commit -m "Update npm dependencies"
+```
+
+Note: in case you feel it, you can also upgrade the npm dependencies to their latest release (major or minor)
+with `make upgrade-npm-deps`
+
+### 1. Prepare your release
+
+You should start to create a branch that follows the pattern `release/v<X.Y.Z>`
+
+Update the file `Changelog.md` and the different `package.json` with the corresponding version:
+
+```bash
+version=v0.2.0
+./ui_release.sh --release "${version}"
+cd ui/
+npm install
+git add ./ui/package-lock.json ./**/package.json
```

Comment:
Could there be an npm script or command in Makefile for this eventually?

Reply:
I managed to add it to the Makefile in the PR https://github.com/perses/perses/pull/336.
I will change this procedure there with the Makefile command.

Spoiler alert it will be like that: 

```bash
make release version="v0.2.0"
```